src/bench/bls.cpp

```diff
@@ -0,0 +1,361 @@
+// Copyright (c) 2018 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "bench.h"
+#include "random.h"
+#include "crypto/bls_worker.h"
+#include "utiltime.h"
+
+#include <iostream>
+
+CBLSWorker blsWorker;
+
+void CleanupBLSTests()
+{
+    blsWorker.Stop();
+}
+
+static void BuildTestVectors(size_t count, size_t invalidCount,
+                             BLSPublicKeyVector& pubKeys, BLSSecretKeyVector& secKeys, BLSSignatureVector& sigs,
+                             std::vector<uint256>& msgHashes,
+                             std::vector<bool>& invalid)
+{
+    secKeys.resize(count);
+    pubKeys.resize(count);
+    sigs.resize(count);
+    msgHashes.resize(count);
+
+    invalid.resize(count);
+    for (size_t i = 0; i < invalidCount; i++) {
```

Comment:
nit: post-increment here and in a bunch of other loops (not a big deal for simple types but should use pre-increment in general)

Reply:
I usually only do pre-increment on iterators and trust in the compiler to properly optimize it for other types

Reply:
Agree, you can ignore this one :)